.mci.yml

```diff
@@ -745,16 +749,17 @@ buildvariants:
           # signature.
           - name: compile_and_test_with_static_libs
 
-    - name: ubuntu1604-zseries
-      display_name: "s390x Ubuntu 16.04 (MongoDB Latest)"
+    - name: ubuntu1804-zseries
+      display_name: "s390x Ubuntu 18.04 (MongoDB 4.4)"
```

Comment:
So "lastest" for this platform points to 4.3, not 4.4, which is what we want to test. Upgrading the server version to 4.4 fixed the tests ([latest evg build](https://evergreen.mongodb.com/version/5f20a1fbd1fe0740912aa8d4))

Reply:
Hah, ironically the latest is older than the released.

I think we'll want to add a `latest` variant again later, when the latest download does update beyond 4.4. I created CXX-2085 to track that.